frontend/src/components/housing/submission/SubmissionIntakeSchema.ts

```diff
@@ -153,14 +153,6 @@ export const submissionIntakeSchema = object({
     ltsaPIDLookup: string().max(255).nullable().label('Parcel ID'),
     geomarkUrl: string().max(255).label('Geomark web service url')
   }),
-  [IntakeFormCategory.PERMITS]: object({
-    hasAppliedProvincialPermits: string().oneOf(YES_NO_UNSURE_LIST).required().label('Applied permits'),
```

Comment:
I think we still want this bit in, doesn't look like it was removed from the form.